tethysapp/gwdm/templates/gwdm/region_map.html

```diff
@@ -58,27 +60,27 @@
 
 {% url 'gwdm:home' as home_url %}
 
-<li class="{% if request.path == home_url %}active{% endif %}"><a href="{{home_url}}">Go Home</a></li>
+<li class="nav-item{% if request.path == home_url %} active{% endif %}"><a href="{{home_url}}">Go Home</a></li>
```

Comment:
The `active` class should be moved to the `<a>` tag.

For example, this:

```
<li class="nav-item{% if request.path == home_url %} active{% endif %}"><a href="{{home_url}}">Go Home</a></li>
```
should be changed to this:

```
<li class="nav-item"><a class="nav-item{% if request.path == home_url %} active{% endif %}" href="{{home_url}}">Go Home</a></li>
```